### PR TITLE
chore(showcase): Add plenty of fish, bitcoin.com, frame.io and sainsbury's to showcase

### LINKED ISSUE #18613: Add four sites to the site showcase
- https://pof.com/ 🐠 
- https://www.bitcoin.com/ 💸 
- https://frame.io/ 🎥 
- https://www.sainsburys.co.uk/ 🛒 

are using Gatsby, and are missing in our site showcase! Let's add them to the end of https://github.com/gatsbyjs/gatsby/blob/master/docs/sites.yml, and tag them as `featured: true`. 

Detailed instructions here: https://www.gatsbyjs.org/contributing/site-showcase-submissions/

### PR DESCRIPTION
**Pull request by Areadrill**
I had some difficulty finding the companies that built these websites... In the cases where I didn't really find any references, I just left the `built_by` as the company's name. Please tell me if something needs changing.

## Description

This PR adds the four aforementioned companies' websites to the showcase file.

## Related Issues
This fixes #18613 .

### CODE REVIEW

File: docs/sites.yml [hunk 1]
```diff
@@ -7846,3 +7846,49 @@
...
+- title: Bitcoin
+  main_url: https://www.bitcoin.com/
+  url: https://bitcoin.com
+  source_url: https://github.com/bitcoin/bitcoin/
+  description: >
+    One of the largest crypto-currency platforms in the world.
+  categories:
+    - Technology
+    - Finance
+    - Open Source
```
[the remark this answers is not in the document]
Comment (Areadrill) [reply docs/sites.yml]: :man_facepalming: of course, bitcoin.com != the actual bitcoin platform. That was a big mixup on my end... My bad!

**Comment by Areadrill** [comment]
Will work on all of these changes at the end of the day.

File: docs/sites.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (fk) [reply docs/sites.yml]: No worries! 🤗

**Comment by fk** [comment]
🎉🙏 Thanks @Areadrill and @LekoArts! 🤗

**Pull request merged**